openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java

```diff
@@ -874,7 +874,9 @@ public PutResponse<T> updateVote(String updatedBy, UUID entityId, VoteRequest re
             .withTimestamp(System.currentTimeMillis())
             .withCurrentVersion(originalEntity.getVersion())
             .withPreviousVersion(change.getPreviousVersion());
-
+    if (supportsSearch) {
```

Comment:
remove this , already postUpdate checks internally

Reply:
handled in latest commit.